back-end/tests/service_test/productService.test.js

```diff
@@ -0,0 +1,53 @@
+const { getAllProductModels } = require('../../services/productsService')
+const db = require('../../services/dbService')
+
+const productModels = [
+  {
+    id: 1,
+    slug: 'student_1',
+    name: 'Student Loan',
+    duration: '5 years fixed',
+    icon: 'student',
+    title: '3% yearly interest rate',
+    description: 'Fixed low interests for students only',
+    interest: 0.03
+  },
+  {
+    id: 2,
+    slug: 'student_2',
+    name: 'Student Loan',
+    duration: '5 years fixed',
+    icon: 'student',
+    title: '3% yearly interest rate',
+    description: 'Fixed low interests for students only',
+    interest: 0.03
+  },
+  {
+    id: 3,
+    slug: 'student_3',
+    name: 'Student Loan',
+    duration: '5 years fixed',
+    icon: 'student',
+    title: '3% yearly interest rate',
+    description: 'Fixed low interests for students only',
+    interest: 0.03
+  },
+  {
+    id: 4,
+    slug: 'student_4',
+    name: 'Student Loan',
+    duration: '5 years fixed',
+    icon: 'student',
+    title: '3% yearly interest rate',
+    description: 'Fixed low interests for students only',
+    interest: 0.03
+  }
+]
```

Comment:
@amelieykw this object is used multiple times, it should be extracted to a separate file under the tests folder and just use it 2-4 times